lua/astrocommunity/sql/dadbod/init.lua

```diff
@@ -0,0 +1,23 @@
+return {
+  {
+    "kristijanhusak/vim-dadbod-ui",
+    dependencies = "tpope/vim-dadbod",
+    lazy = false,
+  },
+  {
+    "hrsh7th/nvim-cmp",
```

Comment:
```suggestion
    "hrsh7th/nvim-cmp",
    optional = true,
```

We should also probably add the optional key just in case someone has disabled `cmp` this will still work well